Coderabbit-best-practices.html

```
@@ -0,0 +1,2592 @@
+        }
+        .hero h1 {
+          font-size: 1.9rem;
+        }
+        .content h2 {
+          font-size: 1.35rem;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <button class="sidebar-toggle" id="sidebarToggle" aria-label="メニュー切り替え">☰</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)